test/unit/lib/runner/test-runner/test-report-handler-tests.js

```diff
@@ -627,6 +627,11 @@ describe('lib/runner/test-runner/test-report-handler.js', function() {
               'JSON',
               testReportHandler._report,
             ],
+            [
```

Comment:
only one of these events should be happening, either the JSON event or the actionJSON event depending on what configHandler.get returns. This should be two tests, and each test should only expect one call of 'testReportSummaryReadyToWrite'. If this test is passing something must be wrong with the code, or the test itself.